src/vdaf/xof.rs

```diff
@@ -330,6 +350,10 @@ pub struct XofFixedKeyAes128 {
     base_block: Block,
 }
 
+// This impl is only used by Mastic right now. The XofFixedKeyAes128Key impl is used in cases where
+// the base XOF can be reused with different contexts. This is the case in VDAF IDPF computation.
+// TODO: try to remove the duplicated code below. init() It's mostly the same as
```

Comment:
Let's add an issue reference to this
```suggestion
// TODO(#1147): try to remove the duplicated code below. init() It's mostly the same as
```